isis/tests/BundleSolutionInfoTests.cpp

```diff
@@ -0,0 +1,228 @@
+#include <QByteArray>
```

Comment:
I think that this file should be named UnitTestBundleSolutionInfo.cpp according to the convention set by the gtest conversion project (https://github.com/USGS-Astrogeology/ISIS3/wiki/Writing-ISIS3-Tests-Using-Gtest-and-Ctest#test-names).